### ISSUE TITLE
[Link]: New tab screen reader text

### ISSUE THREAD

**Issue by mart-sessman** [TEDI-Design-System/angular#244]
## Problem
When link navigation opens new tab there is missing text for screen readers that this opens in a new tab.

## Solution
Add screen reader text that this link opens a new tab.
